src/Common/src/Interop/Windows/SChannel/Interop.SecPkgContext_ApplicationProtocol.cs

```diff
@@ -21,22 +21,25 @@ internal enum ApplicationProtocolNegotiationExt
         ALPN
     }
 
-    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi)]
-    internal class SecPkgContext_ApplicationProtocol
+    [StructLayout(LayoutKind.Sequential)]
+    internal unsafe class SecPkgContext_ApplicationProtocol
```

Comment:
Does this need `unsafe` at the class scope?  A local `unsafe` block, or member-level, seems more appropriate...